website/src/contribute/release-guide.md

```diff
@@ -1135,6 +1183,20 @@ __NOTE__: Only PMC members have permissions to do it, ping [dev@](mailto:dev@bea
 
 Make sure the download address for last release version is upldaed, [example PR](https://github.com/apache/beam-site/pull/478).
 
+#### Deploy SDK docker images to DockerHub
+TODO(hannahjiang): change link to master branch after #9560 is merged.
+
+* Script: [publish_docker_images.sh](https://github.com/Hannah-Jiang/beam/blob/release_script_for_containers/release/src/main/scripts/publish_docker_images.sh)
```

Comment:
Can you add todo here pointing to master once your changes merged?

Reply:
doesn't L1187 work?

Reply:
sg